ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,381 @@
+                                        አዕላፍ ። ። ።</hi>
+                                </incipit>
+                                <note>According to <bibl><ptr
+                                            target="bm:WendyundKleiner2015WalattaPetros"
+                                            /><citedRange>271-272</citedRange></bibl>, the present
+                                    manuscript has an <cit>
+                                        <quote>alternate ending</quote>
+                                    </cit> in comparison to other witnesses.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2.">
+                                <locus from="147va" to="171vb"/>
+                                <title type="complete" ref="LIT7073TammeraW">Taʾammǝra Walatta
+                                    Ṗeṭros</title>
+                                <incipit xml:lang="gez">
+                                    <locus from="147va" to="148rb"/>
+                                    <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ </hi>፩አምላክ፡ ዘለዓለም።
+                                    ዓዲ፡ ናተሉ፡ ጽሒፈ፡ <hi rend="rubric">ተአምረ፡ ዘገብረት፡ እምነ፡ ቡርክት፡ ወለተ፡
+                                        ጴጥሮስ። ጸሎታ፡ ወ</hi>በረከተ፡ ረድኤታ፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ ዓለም፡ አሜን። ስምዑ፡
+                                    ንንግርክሙ፡ ኃይለ፡ ተአምር፡ ዘኮነ፡ እምድኅረ፡ ዕረፍታ፡ ለእምነ፡ ብፅዕት፡ <hi
+                                        rend="rubric">ወለተ፡ ጴጥሮስ፡</hi> ዘሰማዕነ፡ ወዘርኢነ፡ ዘይትገበር፡ በዕለተ፡
+                                    ተዝካራ፡ ለለጊዜሁ፡ እም፴ዕለት፡ እስከ፡ ፍጻሜ፡ ዓመት፡ ከመ፡ ወደረ፡ በረከተ፡ እግዚአብሔር፡ ላዕለ፡
+                                    ኵሉ፡ ዘኮነ፡ ግብረ፡ ተዝካር፡ ዘይትገበር። ወአሜሃ፡ ተጋብኡ፡ ብዙኃን፡ ሰብእ፡ እምሥራቅ፡
+                                    ወእምዕራብ፡ እምአድባር፡ ወእመካናት፡ ወእምገዳማት፡ ወእምኵሉ፡ አኅጉር፡ እለ፡ አልቦሙ፡ ኍልቊ።
+                                    ቦእምኔሆሙ፡ መምህራን፡ ወቀሳውስት፡ ወዲያቆናት፡ ወመነኮሳት፡ ምስለ፡ ማዕጠንታት፡ ወመስቀላት። ወቦ፡
+                                    እምኔሆሙ፡ ነዳያን፡ ወምስኪናን፡ ወዕውራን፡ ወሐንካሳን፡ እስመ፡ ተሰምዓ፡ ዜናሃ፡ ውስተ፡ ኵሉ፡
+                                    አጽናፈ፡ ዓለም። ወወሀብዎሙ፡ ብዙኃ፡ መብልዐ፡ ወመስቴ፡ በበማዕርጊሆሙ፡ ወበበመጠሆሙ፡ እስከ፡ ጸግቡ፡
+                                    ወአትረፉ። ፩ተአምር፡ ዘኮነ፡ ሶቤሃ። ወይቤሉ፡ ሀለወ፡ ኅዳጥ፡ ኅቡአ፡ መጠነ፡ ፩ኅብስት፡ በዓቢይ፡
+                                    ጋን፡ ዘተረፈ፡ እምነ፡ ተዝካር። ወእምዝ፡ ተረክበ፡ እንዘ፡ ይፈልሕ፡ ወይመልዕ፡ እስከ፡ ላዕሉ፡
+                                    ወይትከዓው። </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="171ra" to="172ra"/>
+                                <title>Supplication on behalf of those who were involved in the book
+                                    production</title>
+                                <incipit xml:lang="gez"> ዘአጽሐፍነ፡ እለ፡ መጻእነ፡ ወእለ፡ ተጋባእነ፡ በዛቲ፡ ዕለት፡
+                                    ለሰሚዓ፡ ዝንቱ፡ ገድላቲሃ፡ ወተአምራቲሃ፡ ለእምነ፡ ቡርክት፡ ወለተ፡ ጴጥሮስ፡ </incipit>
+                                <colophon xml:id="coloph1" xml:lang="gez"><title>Colophon</title>
```

Comment:
as discussed orally and correctly attempted here https://github.com/BetaMasaheft/Manuscripts/pull/2622/files if a colophon belongs to several textual units this must be nested accordingly. in the version https://github.com/BetaMasaheft/Manuscripts/pull/2622/files only a title was missing for the nested unit

Reply:
you are right, I changed it without any good reason!